### PR TITLE
Completed Changelog

### PR DESCRIPTION
**Pull request by anhiga**
After going through all the issues and PRs since 0.6 version, I have added all the changes that seemed important from my point of view. You are the main commit author, @Juanlu001, so you should review if you miss something in the list, or there is something that shouldn't be here. 😄

### CODE REVIEW

**Comment by Juanlu001** [comment]
This is very complete @anhiga, thanks a lot! It's amazing how many things we've done in a few months... :) I will edit this directly instead of reviewing.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+.............
+
+* **Pip packaging**: troublesome dependencies have been released in wheel format,
+  so poliastro can now be installed using pip from all platforms.
+* **Coordinates module**: new module containing transformations between ICRS
+  and body-centered frame, and perifocal to body_centered, :py:mod:`~poliastro.coordinates`.
+* **NEOS package**: a new package has been added to poliastro, :py:mod:`~poliastro.neos`
+  package. It provides several ways of getting NEOs (Near Earth Objects) data from NASA
+  databases, online and offline.
+* **New bodies**: added solar system planets (and Pluto and Moon), to bodies
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: The planets were already there, rewording this.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -1,26 +1,76 @@
...
+  module, as well as rotational elements used in body centered frames.
+* **New constants module**: poliastro has now a :py:mod:`~poliastro.constants` module, with GMs and
+  radii of solar system bodies.
 * **Bodies now have a parent**. It is now possible to specify the attractor
   of a body.
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: Will add a new line after #235 is merged.

**Comment by anhiga** [comment]
Great! Can you add the brand-new `from_relative` method, or do I push another commit?

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -1,26 +1,76 @@
...
+Other highlights:
+.................
+
+* **Added Jupyter examples**: poliastro examples are now available in the
+  documentation as Jupyter notebooks, thanks to `nbsphinx`_.
+* **New Code of Conduct**: poliastro community now has a Code of conduct.
+* **Joined Open Astronomy**: we are now part of `Open Astronomy`_, a 
+  collaboration between open source astronomy and astrophysics projects
+  to share resources, ideas, and to improve code.
+* **Included PEP8 and MyPy checks**: code is checked for errors during CI,
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: This is mostly for developers, removing.

File: docs/source/changelog.rst [hunk 4]
```diff
@@ -1,26 +1,76 @@
...
+  testing both PEP8 and MyPy complience.
+* **Documentation update**: documentation has been updated with new installation
+  ways, propagation and NEOs examples, "refactored" code and images, improved contribution
+  guidelines and intersphinx extension.
+* **Docs in CI**: added docs to continuous integration.
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: Same as above.

File: docs/source/changelog.rst [hunk 5]
```diff
@@ -1,26 +1,76 @@
...
+* **New success stories**: two new success stories has been added to documentation.
+* **Legend plotting**: now label and epoch are in a figure legend.
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: Moving this to a more prominent place, and also ask for community feedback.

File: docs/source/changelog.rst [hunk 6]
```diff
@@ -1,26 +1,76 @@
...
+.. _`Issue #205`: https://github.com/poliastro/poliastro/issues/205
 .. _`Issue #128`: https://github.com/poliastro/poliastro/issues/128
-.. _`#131`: https://github.com/poliastro/poliastro/issues/131
+.. _`Issue #131`: https://github.com/poliastro/poliastro/issues/131
+.. _`Issue #130`: https://github.com/poliastro/poliastro/issues/130
+
+Backward incompatible changes:
+..............................
+
+* **Non-osculating orbits**: removed support for non-osculating orbits.
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: I think the term "non-osculating orbits" is a bit confusing, rewording this.

File: docs/source/changelog.rst [hunk 7]
```diff
@@ -1,26 +1,76 @@
 What's new
 ==========
 
-Unreleased
-----------
+poliastro 0.7.0 - 2017-09-07
```
Comment (Juanlu001) [inline docs/source/changelog.rst]: Probably will add some beta mark or something until we have the final release. Today it will be a beta.

**Comment by Juanlu001** [comment]
Merged manually, closing.

**Pull request closed without merging**